tenable/io/v3/vm/agent_config.py

```diff
@@ -0,0 +1,93 @@
+'''
+Agent Config
+============
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`agent config <agent-config>` API endpoints.
+
+Methods available on ``tio.agent_config``:
+
+.. rst-class:: hide-signature
+.. autoclass:: AgentConfigAPI
+    :members:
+'''
+from typing import Dict
+
+from ..base.endpoints.explore import ExploreBaseEndpoint
+
+
+class AgentConfigAPI(ExploreBaseEndpoint):
+    '''
+    This will contain all methods related to agent config
+    '''
+    # _path: str = 'api/v3/agents'
+    _conv_json: bool = True
+
+    def edit(self, auto_unlink: int = None, software_update: bool = None,
```

Comment:
Please keep agent_id as the first parameter, and also remove the default value. Let's keep it required parameter.

Reply:
Done.

Reply:
@milanaghera-tenable I think agent_id is still third parameter. Could you please move it as the first parameter?

Reply:
resolved.